src/gromacs/CMakeLists.txt

```diff
@@ -259,6 +259,8 @@ a clFFT package, or use the latest GROMACS 2018 point release.")
     endif()
 endif()
 
+target_sources(libgromacs PRIVATE nblib)
```

Comment:
This goes back to what I keep asking about: whether nblib depends on Gromacs or the other way around. gmxapi has the same confusion with naming. We should both try to find a way to distinguish the Gromacs infrastructure that supports the external API from the external API. We should refine the target and directory names at some point, but it definitely warrants some internal documentation in the near term.

Reply:
It turns out that this line is not needed at all. Removed in the latest commit. 

Note that this change does not intend to allow compiling programs against nb-lib, but only to allow ease migration in a future patch that does so.